.github/workflows/main.yml

```diff
@@ -33,7 +33,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [Linux, macOS, Windows]
```

Comment:
I this because this breaks windows, or because we want to stop supporting windows for a bit?  I haven't been following CI stuff very much.

Reply:
The cargo windows build has race conditions due to our use of `cxx-build`. I tried to do some workarounds but none of them seem to work.